Gems/ROS2/Code/Source/Gripper/GripperActionServer.cpp

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root
+ * of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "Utils.h"
+
+#include "GripperActionServer.h"
+#include <AzCore/Serialization/EditContext.h>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzFramework/Components/TransformComponent.h>
+
+namespace ROS2
+{
+    void GripperActionServer::Activate()
+    {
+        auto ros2Node = ROS2Interface::Get()->GetNode();
+
+        actionServer = rclcpp_action::create_server<GripperCommand>(
+            ros2Node,
+            m_gripperActionServerName.data(),
```

Comment:
This is not respecting the namespaces. 
Imagine you serialize a gripper entity and the user changes ROS2Frame namespace. This server would not respect the change. Please correct to query ROS2Frame for namespace.

Reply:
Also add ROS2Frame as required service.